js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -82,7 +82,22 @@ export default class ExtensionRenderer extends React.Component {
             newChildren = this.props.children;
         }
 
-        return React.createElement(this.props.wrappingElement, null, newChildren);
+        const {
+            className,
+            extensionPoint
+        } = this.props;
+
+        const classNames = ['ExtensionPoint', extensionPoint.replace(/\.+/g,'-')];
+        
+        if (className) {
+            classNames.push(className);
+        }
+
+        const newProps = {
+            className: classNames.join(' ')
+        };
+
+        return React.createElement(this.props.wrappingElement, newProps, newChildren);
```

Comment:
I personally would have extracted `this.props.wrappingElement` as well in the earlier statement, but I guess personal preference